employees/forms.py

```diff
@@ -32,13 +32,13 @@ def __init__(self, queryset: QuerySet, *args: Any, **kwargs: Any) -> None:
 
 class DurationInput(TextInput):
     def format_value(self, value: Optional[str]) -> str:
-        if value is not None:
+        if value is not None and value != ":":
             return timedelta_to_string(parse_duration(value))
         return ""
 
 
 class DurationFieldForm(forms.DurationField):
-    widget = DurationInput
+    widget = DurationInput(attrs={"data-mask": "99:99", "placeholder": "H:MM"})
```

Comment:
What does it mean that this field has `99:99` mask?

Reply:
https://dobsondev.com/2017/04/14/using-jquery-mask-to-mask-form-input/
> * ‘0’: {pattern: /\d/}
> * ‘9’: {pattern: /\d/, optional: true}

Reply:
So shouldn't it be `90:00`? Which means that you should have at least one digit before `:` and two after. This way you can type in values like `12:00`, `8:00` and `0:30`.  Or maybe I have misunderstood this completely.

Reply:
quite frankly, it must be `09:99` (checked empirically;)).